### PR TITLE
Migrate from vscode.previewHtml command to new VS Code WebView API

### PR DESCRIPTION
**Pull request by SounD120**
#827 Migrating to new WebView API

### CODE REVIEW

File: src/extension/exponent/exponentPlatform.ts [hunk 1]
```diff
@@ -68,7 +71,12 @@ export class ExponentPlatform extends GeneralMobilePlatform {
                     return Q.reject<string>(reason);
                 })
                 .then(exponentUrl => {
-                    vscode.commands.executeCommand("vscode.previewHtml", vscode.Uri.parse(exponentUrl), 1, "Expo QR code");
+                    let exponentPage = vscode.window.createWebviewPanel("Expo QR Code", "Expo QR Code", vscode.ViewColumn.Two, {
+                         enableScripts: true,
```
Comment (ruslan-bikkinin) [inline src/extension/exponent/exponentPlatform.ts]: Please clarify, why do we need to enable scripts for this page? According to this docs, we shouldn't allow this unless it's really needed:
https://code.visualstudio.com/docs/extensions/webview#_limit-capabilities

File: src/extension/qrCodeContentProvider.ts [hunk 2]
```diff
@@ -2,13 +2,13 @@
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
 import * as qr from "qr-image";
-import { TextDocumentContentProvider, Uri, CancellationToken } from "vscode";
```
Comment (ruslan-bikkinin) [inline src/extension/qrCodeContentProvider.ts]: Please clarify why did you make this change? Is the CancellationToken is redundant?

Comment (SounD120) [reply src/extension/qrCodeContentProvider.ts]: Yes, it's not used now

File: src/extension/exponent/exponentPlatform.ts (hunk 1 above)
Comment (SounD120) [reply src/extension/exponent/exponentPlatform.ts]: Looks like webpage is working without this options, so I deleted them.

**Pull request merged**